packages/rollup-core/src/app/data/producers/l1-chain-data-persister.ts

```diff
@@ -29,20 +29,23 @@ export class L1ChainDataPersister extends ChainDataProcessor {
    * @param l1Provider The provider to use to connect to L1 to subscribe & fetch block / tx / log data.
    * @param logHandlerContexts The collection of LogHandlerContexts that uniquely identify the log events
    *        to be handled and the function that processes their data and inserts them into the RDB.
+   * @param earliestBlock The earliest block to sync.
```

Comment:
```suggestion
   * @param earliestBlock The earliest L1 block to sync.
```

Reply:
It's in the L1 Chain Data Persister, so probably implied, but will update 👍